src/crons/nft/nft.cron.service.ts

```diff
@@ -64,9 +64,11 @@ export class NftCronService {
           continue;
         }
 
-        const isAssetUploaded = await this.nftAssetService.isAssetUploaded(nft.identifier, nft.media[0]);
-        if (!isAssetUploaded) {
-          needProccessNfts.push(nft);
+        for (const media of nft.media) {
+          const isAssetUploaded = await this.nftAssetService.isAssetUploaded(media);
+          if (!isAssetUploaded) {
+            needProccessNfts.push(nft);
```

Comment:
If an NFT have more than one media files it will be processed more than one time